dashboard.py

```diff
@@ -17,23 +18,30 @@
 async def results(request):
     """Fetch remote results forward on."""
 
+    def _derive_mapping_code(response, mapped_totals):
+        """Consumes a list of dictionaries and returns a mapping of 2 char CC-SS (Country State)
+        keys and total submissions for each."""
+        values = response['values']
+        code = '{0}-{1}'.format([x['value'].lower() for x in values if x['label'] == 'country_code'][0], [x['value'].lower() for x in values if x['label'] == 'state_code'][0])  # noqa
```

Comment:
I don't have any other ideas and it is functional. It is really long wouldn't be bad to break it up to a few lines.